### PR TITLE
more opam keywords (MCA-dev meeting of 2023-12-21)

### PR DESCRIPTION
**Pull request by affeldt-aist**
##### Motivation for this change

as discussed during the meeting

##### Things done/to do

~~- [ ] added corresponding entries in `CHANGELOG_UNRELEASED.md`~~

~~- [ ] added corresponding documentation in the headers~~

##### Compatibility with MathComp 2.0

- [x] I added the label `TODO: HB port` to make sure someone ports this PR to
      the `hierarchy-builder` branch **or** I already opened an issue or PR (please cross reference).

##### Automatic note to reviewers

Read [this Checklist](https://github.com/math-comp/math-comp/wiki/Checklist-for-creating-and-review-PRs) and put a milestone if possible.

### CODE REVIEW

File: coq-mathcomp-analysis.opam [hunk 1]
```diff
@@ -27,13 +27,26 @@ depends: [
 tags: [
   "category:Mathematics/Real Calculus and Topology"
   "keyword:analysis"
+  "keyword:extended real number"
+  "keyword:filter"
+  "keyword:Cantor"
   "keyword:topology"
-  "keyword:real numbers"
+  "keyword:real number"
```
Comment (proux01) [inline coq-mathcomp-analysis.opam:34]: Why not keep the plural?

**Review by proux01 — APPROVED** [review]
LGTM

File: coq-mathcomp-analysis.opam (hunk 1 above)
Comment (affeldt-aist) [reply coq-mathcomp-analysis.opam:34]: There are actually a couple of entries where plural would also seem appropriate to me (e.g., set/sets, function/functions, derivative/derivatives) but I couldn't decide a rule so I put everything singular and read each entry XYZ as "the notion of XYZ".

Comment (proux01) [reply coq-mathcomp-analysis.opam:34]: I'd rather keep `real numbers` and `sets` but that's not a strong opinion.

**Pull request merged**